### PR TITLE
feat: support Powershell 5

### PR DESCRIPTION
**Pull request by PIYUSH194**
### Prerequisites
- [ ] Fix iswindows  variable found in windows powershell

### Description

If iswindows variable not available in windows powershell it will through chmod command not found.
not applicable on powershell 6 and greater

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
Wouldn't it semantically make more sense to check for PSVersion > 5?

**Comment by JanDeDobbeleer** [comment]
Given that support for 6 isn't available, I'd opt for lowering the version number to 5 if this is all that's needed to support 5. I suppose you're using this code in PS5?

**Comment by PIYUSH194** [comment]
It works fine in PS5.. why no support for PS5 ?

**Comment by JanDeDobbeleer** [comment]
That was my question indeed. Allow me to extend this properly tomorrow.

File: packages/powershell/oh-my-posh/oh-my-posh.psm1 [hunk 1]
```diff
@@ -20,7 +20,7 @@ function Get-PoshCommand {
 }
 
 # Set the right binary to executable before doing anything else
-if (!$IsWindows) {
+if (!$IsWindows  -and (Get-Item "Variable:*IsWindows*")) {
```
Comment (JanDeDobbeleer) [inline packages/powershell/oh-my-posh/oh-my-posh.psm1]: Can we change this to: `if ($PSVersionTable.PSEdition -eq "Core" -and !$IsWindows)`
Which means we can change `oh-my-posh.psm1` to include 5 as well within the scope of this PR and use `feat: support Powershell 5` as a commit message.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: packages/powershell/oh-my-posh/oh-my-posh.psm1 (hunk 1 above)
Comment (PIYUSH194) [reply packages/powershell/oh-my-posh/oh-my-posh.psm1]: How can i add only last commit to this pull request? Or you can close this request and add it manually

Comment (JanDeDobbeleer) [reply packages/powershell/oh-my-posh/oh-my-posh.psm1]: If you need to know how, learning to rebase is key here -> https://www.algolia.com/blog/master-git-rebase/
I can always pick this change, you stay the author in that case, let me know what would work for you.

Comment (PIYUSH194) [reply packages/powershell/oh-my-posh/oh-my-posh.psm1]: Just pick the change that will be good.

**Pull request closed without merging**